package.json

```diff
@@ -63,18 +70,14 @@
     "babel-runtime": "^6.3.19",
     "bootstrap": "^3.3.6",
     "classnames": "^2.1.5",
+    "cloudgov-style": "git+https://git@github.com/18F/cg-style.git#master",
```

Comment:
It makes me a bit nervous to pull straight off of `#master` since that can change without us being aware and break the build if the project changes significantly. This happened to me on USWDS.

Could we start doing releases on cloudgov-style or is that too much overhead?